dss-core/src/test/java/io/github/ztkmkoo/dss/core/actor/rest/DssRestAuthActorTest.java

```diff
@@ -0,0 +1,66 @@
+package io.github.ztkmkoo.dss.core.actor.rest;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+
+import akka.actor.testkit.typed.javadsl.TestProbe;
+import akka.actor.typed.ActorRef;
+import io.github.ztkmkoo.dss.core.actor.AbstractDssActorTest;
+import io.github.ztkmkoo.dss.core.message.security.DssAuthCommand;
+import io.github.ztkmkoo.dss.core.message.security.DssAuthenticationCommandRequest;
+import io.github.ztkmkoo.dss.core.message.security.DssAuthenticationCommandResponse;
+import io.github.ztkmkoo.dss.core.message.security.DssAuthorizationCommandRequest;
+import io.github.ztkmkoo.dss.core.message.security.DssAuthorizationCommandResponse;
+
+public class DssRestAuthActorTest extends AbstractDssActorTest {
+
+	 private static final TestProbe<DssAuthCommand> probe = testKit.createTestProbe();
+	 private static final ActorRef<DssAuthCommand> restAuthActorRef = testKit.spawn(DssRestAuthActor.create(testUserList()), "rest-auth");
+	 private static String testToken;
+	 
+	 private static Map<String, String> testUserList() {
+		 final Map<String, String> userList = new HashMap<>();
+		 userList.put("testID", "testPassword");
+		 
+		 return userList;
+	 }
+	 
+	 @Test
+	 void handlingDssAuthenticationCommandRequest() {
+		 restAuthActorRef.tell(DssAuthenticationCommandRequest
+				 .builder()
+				 .userID("testID")
+				 .userPassword("testPassword")
+				 .token(probe.ref())
+				 .build());
+		 
+		 DssAuthenticationCommandResponse response = probe.expectMessageClass(DssAuthenticationCommandResponse.class);
+		 testToken = response.getToken();
+		 
+		 assertTrue(true);
+		 assertNotNull(response);
+	 }
+	 
+	 @Test
+	 void handlingDssAuthorizationCommandRequest() {
+		 restAuthActorRef.tell(DssAuthorizationCommandRequest
+				 .builder()
+				 .userID("testID")
+				 .token(testToken)
+				 .valid(probe.ref())
+				 .build());
+		 
+		 DssAuthorizationCommandResponse response = probe.expectMessageClass(DssAuthorizationCommandResponse.class);
+		 
+		 assertTrue(true);
+		 assertNotNull(response);
+	     assertEquals("true", response.getValid());
```

Comment:
@RulLu16 
Hi, i found the ci was failed here. It looks when test `handlingDssAuthorizationCommandRequest` was exected first causes the test failed.

I suggest:
- ordering the tests or make sure testToken not null(in this test, get the token before you authorization)
- improve your DssRestAuthActor that handling when token is null. => it is better that response valid false when token is invald or token is null

Reply:
Thank you for review! I agree with your suggestion about ordering the tests. 
And also, I found another error about signature jwts. So trying to fix it.